### PR TITLE
Email Migration Docs

### PR DESCRIPTION
**Pull request by juandesi**

### CODE REVIEW

File: connectors/v/latest/email-gmail.adoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+toc::[]
+
+[[short_description]]
+The examples here provide guidance on setting up a Gmail connection using the Email connector over
+each of the available protocols.
+
+== Enable Less Secure clients
+
+Google might block (by default) third-party (less secure) mail clients.
```
Comment (marianogonzalez) [inline connectors/v/latest/email-gmail.adoc]: remove the `(less secure)`. We don't want clients asking why is the connector not secure enough

File: connectors/v/latest/_toc.adoc [hunk 2]
```diff
@@ -28,7 +28,7 @@
 *** link:/db-connector-datatypes-ref[Database Connector Data Types Reference]
 *** link:/db-connector-transactions-ref[Database Transactions Reference]
 *** link:/db-connector-execute-script-ref[Execute Script Reference]
-** link:/email-about-the-email-connector[Email Connector]
+** link:/email-connector[Email Connector]
```
Comment (marianogonzalez) [inline connectors/v/latest/_toc.adoc]: We usually live the TOC part to @dukesphere . Stacey, can you review this please?

File: connectors/v/latest/_toc_email.adoc [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // Email Connector TOC Include for _toc.adoc
 ** link:/connectors/email-connector[Email Connector]
-*** link:/connectors/email-about-the-email-connector[About the Email Connector]
+*** link:/connectors/email-connector[About the Email Connector]
```
Comment (marianogonzalez) [inline connectors/v/latest/_toc_email.adoc]: We usually live the TOC part to @dukesphere . Stacey, can you review this please?

File: connectors/v/latest/email-connector.adoc [hunk 4]
```diff
@@ -46,10 +46,15 @@ This example sets a configuration for the POP3S gmail mailbox.
...
 When listing emails, users are often not only interested in the email's content, but in its metadata, as well (for example, the subject, from addresses, to addresses, received date, and so on). These attributes might vary depending
-on the type of configuration that you are using. The email connector uses Mule message attributes to access this information, while the message pyayload always contains the email's content.
+on the type of configuration that you are using. The email connector uses Mule message attributes to access this information, while the message payload always contains the email's content.
 
 For details on the structure of email attributes, please read the link:email-documentation[Email Module Documentation Reference].
 
 == See Also
 * link:email-list[To List emails]
 * link:email-send[To Send emails]
+* link:email-gmail[Connecting with Gmail]
```
Comment (marianogonzalez) [inline connectors/v/latest/email-connector.adoc]: what about fixing the rest of the email documentation??? It's in pretty bad shape

File: connectors/v/latest/email-gmail.adoc (hunk 1 above)
Comment (juandesi) [reply connectors/v/latest/email-gmail.adoc]: fixed

**Pull request merged**